lms/djangoapps/verified_track_content/tests/test_models.py

```diff
@@ -183,3 +201,12 @@ def test_unenrolled(self):
         self.assertEqual(DEFAULT_COHORT_NAME, get_cohort(self.user, self.course.id, assign=False).name)
         self._upgrade_to_verified()
         self.assertEqual(VERIFIED_COHORT_NAME, get_cohort(self.user, self.course.id, assign=False).name)
+
+    def test_custom_cohort_name(self):
```

Comment:
Please name "test_custom_verified_cohort_name", as I am adding tests/ability for default group name to be specified as well.